src/Grid/grid.jl

```diff
@@ -131,6 +157,11 @@ reference_facets(::Type{<:AbstractRefShape})
 @inline reference_facets(refshape::Type{<:AbstractRefShape{2}}) = reference_edges(refshape)
 @inline reference_facets(refshape::Type{<:AbstractRefShape{3}}) = reference_faces(refshape)
 
+@inline reference_faces(::AbstractCell{refshape}) where refshape <:AbstractRefShape = reference_faces(refshape)
+@inline reference_edges(::AbstractCell{refshape}) where refshape <:AbstractRefShape = reference_edges(refshape)
+@inline reference_vertices(::AbstractCell{refshape}) where refshape <:AbstractRefShape = reference_vertices(refshape)
+@inline reference_facets(::AbstractCell{refshape}) where refshape <:AbstractRefShape = reference_facets(refshape)
```

Comment:
These seem untested, I don't think we use them, but rather use `reference_faces(refshape)` etc. directly, see e.g. the `vertices` function. 
If introduced here, should be used there, but not really needed if only required there?

Still, if we want this, then
```suggestion
@inline reference_faces(::AbstractCell{refshape}) where refshape = reference_faces(refshape)
@inline reference_edges(::AbstractCell{refshape}) where refshape = reference_edges(refshape)
@inline reference_vertices(::AbstractCell{refshape}) where refshape = reference_vertices(refshape)
@inline reference_facets(::AbstractCell{refshape}) where refshape = reference_facets(refshape)
```
(already restricted in `AbstractCell` [ref](https://github.com/Ferrite-FEM/Ferrite.jl/blob/c33a191f618fa9dfb2d5b5efb3c6eae8000ba6d1/src/Ferrite.jl#L60))